src/module/data/item/test.ts

```diff
@@ -2,21 +2,55 @@ import { SystemDataModel } from "@data/abstract.ts"
 import { ItemDataModel } from "./base.ts"
 import { BasicInformationTemplate } from "./templates/basic-information.ts"
 
-class TestSystemData extends SystemDataModel<TestSystemSchema> {}
+declare class TestSystemData extends SystemDataModel<TestSystemSchema> {
+	constructor(...args: any[])
```

Comment:
This constructor is actually necessary. Without it you'll get a "base constructor must all return the same type" style error. This happens because TypeScript has a number of preconceptions about what can safely be mixed, that being that at runtime it looks like `constructor(...args: any[]) { super(...args) }`